### DISCUSSION TITLE
Error while running python ultralytics_stream_example.py

### DISCUSSION THREAD

**Discussion by MHD-HD98** [roboflow/supervision#1213] [Q&A] [answered]
Im running the ultralytics_stream_example.py and getting error  

Encountered inference error: 'list' object has no attribute 'image'                    inference_pipeline.py:822
                             Traceback (most recent call last):
                               File
                             "c:\Users\Lenovo\Desktop\vid\supervision\examples\time_in_zone\.venv\lib\site-packages
                             \inference\core\interfaces\stream\inference_pipeline.py", line 794, in
                             _execute_inference
                                 predictions = self._on_video_frame(video_frames)
                               File
                             "C:\Users\Lenovo\Desktop\vid\supervision\examples\time_in_zone\ultralytics_stream_exam
                             ple.py", line 92, in inference_callback
                                 results = model(frame.image, verbose=False, conf=confidence, device=device)[0]

Attached is the screenshot.

![error](https://github.com/roboflow/supervision/assets/69430040/92b893d5-ff70-4797-93e0-ed3c3041cd4d)

**Comment by SkalskiP** [comment, marked as answer]
Hi, @MHD-HD98! 👋🏻 Looks like you installed an incorrect version of `inference`. That demo requires `inference==0.9.17`. Braking API change accused in later versions.

**Reply by MHD-HD98** [reply]
Hi, @SkalskiP It worked! Thanks for the fix

**Reply by SkalskiP** [reply]
Pleasure!
